influxdb/content.md

```diff
@@ -2,15 +2,15 @@
 
 InfluxDB is a time series database built from the ground up to handle high write and query loads. InfluxDB is meant to be used as a backing store for any use case involving large amounts of timestamped data, including DevOps monitoring, application metrics, IoT sensor data, and real-time analytics.
 
-[InfluxDB Documentation](https://docs.influxdata.com/influxdb/latest/)
+[InfluxDB 1.8 documentation](https://docs.influxdata.com/influxdb/v1.8/)
 
 %%LOGO%%
 
 ## Updating latest to 2.0
 
 In the future, the latest tag for this image will point to the latest released implementation of influxdb 2.0. This will happen when the first general available release for 2.0 exists. If you are using the `latest` tag for any production or development purposes, please update your development environment to reference the `1.8` tag.
```

Comment:
Because this page currently refers to 1.8 and earlier, updated links to 1.8 docs (and removed `latest`) tag.

Now that OSS GA has been released, does it make sense to update this statement to say:
_As of the InfluxDB 2.0 OSS (open source) general availability release, the `latest` tag points to the latest 2.0 release. We are currently working to  If you are using the `latest` tag for any production or development purposes, update your development environment to reference the `1.8` tag._